### DISCUSSION TITLE
Contributions Welcome?

### DISCUSSION THREAD

**Discussion by deathbreakfast** [patternfly-yew/patternfly-yew#8] [Q&A] [answered]
I've been using ybc for a while and it's not actively maintained. Last two Yew updates slowed me down quite a bit.

I threw up a couple pull requests to add a component. Since I'm going to be building several projects with Yew, mind if start creating issues and PR's for things as I come across them?

**Comment by ctron** [comment, marked as answer]
I don't mind at all! I would really appreciate this actually.

This project is mostly drive by the demand we have ourselves. So, there is no plan at the moment to map out all the components that patternfly supports. But if stuff is missing I would definitely appreciate an issue or a PR.
